relay/relay.go

```diff
@@ -383,6 +392,14 @@ func (rs *Relay) GetPayload(ctx context.Context, m *structs.MetricGroup, payload
 		"processingTimeMs": time.Since(tStart).Milliseconds(),
 	})
 
+	go func() {
```

Comment:
Not a goroutine here, just write to the channel (can be hidden inside api). Also what we would like to save is a raw payload that someone sent to us. So that we can capture code failures

Reply:
If the data-exporter buffer is full, or the writers are working on it, then the `GetPayload` response latency increases. That should never happen. Exporting data should not affect to the time it takes to respond `GetPayload`

Reply:
Below we are already storing the payload in goroutine after publishing it to maybe reuse that one, or move data api to the common one.